src/Styleguide/Components/FollowArtistPopover/index.tsx

```diff
@@ -0,0 +1,87 @@
+import { FollowArtistPopover_suggested } from "__generated__/FollowArtistPopover_suggested.graphql"
+import { FollowArtistPopoverQuery } from "__generated__/FollowArtistPopoverQuery.graphql"
+import { ContextConsumer, ContextProps } from "Artsy/SystemContext"
+import React from "react"
+import { createFragmentContainer, graphql, QueryRenderer } from "react-relay"
+import styled from "styled-components"
+import { FollowArtistPopoverRowFragmentContainer as FollowArtistPopoverRow } from "./FollowArtistPopoverRow"
+
+const Container = styled.div``
+
+interface Props extends ContextProps {
+  suggested: FollowArtistPopover_suggested
+}
+
+class FollowArtistPopover extends React.Component<Props, null> {
```

Comment:
Can you convert this to a SFC? As we're preparing for the new React we should only use classes when state or tracking is required.

Reply:
Yep, totally! 👍